modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/ItReplicaLifecycleTest.java

```diff
@@ -659,97 +656,6 @@ public void testScanCloseReplicaRequest(TestInfo testInfo) throws Exception {
         assertDoesNotThrow(tx::commit);
     }
 
-    @Test
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-24558")
-    public void testCatalogCompaction(TestInfo testInfo) throws Exception {
-        // How often we update the low water mark.
-        long lowWatermarkUpdateInterval = 500;
-        updateLowWatermarkConfiguration(lowWatermarkUpdateInterval * 2, lowWatermarkUpdateInterval);
-
-        // Prepare a single node cluster.
-        startNodes(testInfo, 1);
-        Node node = getNode(0);
-
-        List<Set<Assignment>> assignments = PartitionDistributionUtils.calculateAssignments(
-                nodes.values().stream().map(n -> n.name).collect(toList()), 1, 1);
-
-        List<TokenizedAssignments> tokenizedAssignments = assignments.stream()
-                .map(a -> new TokenizedAssignmentsImpl(a, Integer.MIN_VALUE))
-                .collect(toList());
-
-        placementDriver.setPrimary(node.clusterService.topologyService().localMember());
-        placementDriver.setAssignments(tokenizedAssignments);
-
-        forceCheckpoint(node);
-
-        String zoneName = "test-zone";
-        createZone(node, zoneName, 1, 1);
-        DistributionZonesTestUtil.getZoneId(node.catalogManager, zoneName, node.hybridClock.nowLong());
```

Comment:
Just a comment: very weird that the statement didn't used but was presented.

Reply:
Yep, that's my fault actually.

Reply:
you are welcome